Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -124,10 +126,43 @@ final class PledgeDescriptionCell: UITableViewCell, ValueCell {
       view.heightAnchor.constraint(equalToConstant: Layout.SpacerView.height).isActive = true
       self.descriptionStackView.insertArrangedSubview(view, at: 3)
     }
-
     _ = ([self.descriptionStackView], self.rootStackView)
       |> ksr_addArrangedSubviewsToStackView()
   }
+
+  // MARK: - Binding
+
+  internal override func bindViewModel() {
+    super.bindViewModel()
+
+    self.dateLabel.rac.text = self.viewModel.outputs.estimatedDeliveryText
+
+    self.viewModel.outputs.presentTrustAndSafety
+      .observeForUI()
+      .observeValues { [weak self] in
+        guard let _self = self else { return }
+        self?.delegate?.pledgeDescriptionCellDidPresentTrustAndSafety(_self)
```

Comment:
Man, this seems really confusing (using two `self` references)...wouldn't this suffice?
```
guard let self = self else { return }
self.delegate.pledgeDescriptionCellDidPresentTrustAndSafety(self)
```